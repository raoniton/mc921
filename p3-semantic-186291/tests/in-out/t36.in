class Main {
    int[] v = {1, 2, 3, 4};
    public static void main(String[] args) {
        char x = '3';
        for (int i = 1; i <= 4; i=i+1)
            v[x] = i;
    }
}