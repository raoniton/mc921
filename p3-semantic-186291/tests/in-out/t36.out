SemanticError: Array dimension must be of type(int), not type(char) @ 6:13
